class7-week3/actions/markAsDone.js

```diff
@@ -0,0 +1,23 @@
+const Todo = require('../models/todo')
+const deserializeTodo = require('../util/deserializeTodo')
+
+module.exports = function markAsDone(request, response) {
+
+  const id = request.params.id
+  const todo = deserializeTodo(request, response)
```

Comment:
This got a little tricky to debug... Same comments apply to markAsNotDone.js

You should comment out lines 7 and 8. Those are only useful when you're extracting a todo from the body of a POST/PUT request. We aren't doing that here, so there's nothing to deserialize.